### PR TITLE
Bug: send correct currency code to Adyen

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Store currency code by country

- Send correct currency code to Adyen

## Justify why they are needed

- For some reason we sent locale and this caused errors

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3808** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3808?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3808?utm_source=stack-comment).

File: apps/store/src/services/adyen/useGetAdyenCheckout.ts [hunk 1]
```diff
@@ -31,11 +31,11 @@ export const useGetAdyenCheckout = (): ((options: CoreOptions) => AdyenCheckoutA
           'applepay',
         ],
 
-        paymentMethodsConfiguration: getPaymentMethodsConfiguration(locale, countryCode),
+        paymentMethodsConfiguration: getPaymentMethodsConfiguration(currencyCode, countryCode),
```
Comment (guilhermespopolin) [inline apps/store/src/services/adyen/useGetAdyenCheckout.ts:34]: Nice catch. I guess that wouldn't happen if we had a object/enum with the possible/supported currency right? Do you think adding that is worth it?

**Review by guilhermespopolin — APPROVED** [review]
Nice catch

File: apps/store/src/services/adyen/useGetAdyenCheckout.ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/services/adyen/useGetAdyenCheckout.ts:34]: nah not really, adyen is being removed in a few months also

**Pull request merged**